test/e2e/network/service.go

```diff
@@ -930,6 +930,109 @@ var _ = common.SIGDescribe("Services", func() {
 		validateEndpointsPortsOrFail(cs, ns, serviceName, portsByPodName{})
 	})
 
+	ginkgo.It("should be updated after adding or deleting ports ", func() {
+		// repacking functionality is intentionally not tested here - it's better to test it in an integration test.
+		serviceName := "edit-port-test"
+		ns := f.Namespace.Name
+		jig := e2eservice.NewTestJig(cs, ns, serviceName)
+
+		defer func() {
+			err := cs.CoreV1().Services(ns).Delete(context.TODO(), serviceName, metav1.DeleteOptions{})
+			framework.ExpectNoError(err, "failed to delete service: %s in namespace: %s", serviceName, ns)
+		}()
```

Comment:
you don't need this defer, the e2e framework cleans up the namespace and the garbage collector delets the objects on it

Reply:
Got it, thanks!